src/PAModel/Checksum/ChecksumMaker.cs

```diff
@@ -157,12 +163,12 @@ private void AddJsonFile(string filename, byte[] bytes)
 
                 var ctx = new Context { Filename = filename };
 
-                using (var hash = IncrementalHash.CreateHash(HashAlgorithmName.SHA256))
+                using (var hash = new T())
```

Comment:
We can pass in a the real hash alg (sha256) or a "debug" one (which just collects the normalized values)